applications/salk-portal/src/pages/ExperimentsPage.tsx

```diff
@@ -78,13 +80,26 @@ const ExperimentsPage = () => {
     const [experiment, setExperiment] = useState(null)
     const [selectedAtlas, setSelectedAtlas] = useState(getDefaultAtlas());
     const [subdivisions, setSubdivisions] = useState(getSubdivisions(selectedAtlas));
-    const [populations, setPopulations] = useState({});
+    const [populations, setPopulations] = useState({} as any);
+    const [sidebarPopulations, setSidebarPopulations] = useState({} as any);
```

Comment:
Using a new state variable so that the widgets don't get updated but only the sidebar